disboardreminder/disboardreminder.py

```diff
@@ -503,15 +480,16 @@ async def on_message_without_command(self, message: discord.Message):
         data = await self.config.guild(guild).all()
         if not data["channel"]:
             return
+        
         clean = data["clean"]
         my_perms = channel.permissions_for(guild.me)
-
-        if embed := self.validate_success(message):
+        
+        if self.validate_success(message):
             last_bump = data["nextBump"]
             if last_bump and last_bump - message.created_at.timestamp() > 0:
                 return
-            await self.respond_to_bump(data, bump_channel, message, embed)
-        elif my_perms.manage_messages and clean and channel == bump_channel:
+            await self.respond_to_bump(data, bump_channel, message)
+        elif (my_perms.manage_messages or message.guild.me.guild_permissions.administrator) and clean and channel == bump_channel:
```

Comment:
This second check is redundant, since being administrator guarantees the bot will have manage message perms.